README.md

```diff
@@ -75,8 +75,25 @@ Installation process for the setting up Wordpress on a local environment.
       brew services start selenium-server-standalone
 
       brew install chromedriver
+      brew services start chromedriver
+
       brew install phantomjs
 
+- **Install WordPress**
+	
+	  php -S localhost:8222 -t ./public
+
+  Open `http://localhost:8222` in browser
+
+  Click "Continue" and then fill in the following fields:
+
+  	Site Title: chap-press
+  	Username: chappress
+  	Password: password (Checkbox "Confirm Password")
+  	Your Email: chappress@gmail.com
+
+  Click "Install Wordpress"
+
```

Comment:
This keeps coming back but I really don't want this. We need to automate this. New dev should be able to go directly from command-line install steps to running tests.